ledger/ledger_test.go

```diff
@@ -1490,7 +1490,7 @@ func triggerTrackerFlush(t *testing.T, l *Ledger, genesisInitState ledgercore.In
 	l.trackers.lastFlushTime = time.Time{}
 	l.trackers.mu.Unlock()
 
-	const timeout = 3 * time.Second
+	const timeout = 2600 * time.Millisecond
```

Comment:
This change is a spitball. Looking at https://github.com/algorand/go-algorand/pull/5379/files I'm inclined to revert it.